tests/x11/disable_screensaver.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Ensure screensaver is working and then disable it
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use x11utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'x11';
+    ensure_unlocked_desktop;
+    assert_screen "generic-desktop";
+    # wait for screensaver to start
+    assert_screen [qw(screenlock screenlock-password)], 700;
+    ensure_unlocked_desktop;
+
+    mouse_hide(1);
+    turn_off_screensaver;
+
+    # TODO: remove this
+    # ensure that the screensaver really got disabled
+    sleep 700;
```

Comment:
Same here.

Reply:
When reconfiguration wouldn't have worked, I wouldn't detect it then.

Reply:
So reconfiguring both to say, 10-60 seconds should allow you to detect the case when it didn't work.

Reply:
1m